src/index.html

```diff
@@ -5,8 +5,302 @@
     <meta name="viewport" content="width=device-width, initial-scale=1.0">
     <title>Catalog</title>
     <link rel="stylesheet" href="./styles/main.scss">
+    <link rel="preconnect" href="https://fonts.googleapis.com">
+    <link rel="preconnect" href="https://fonts.gstatic.com">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@400;500;700&display=swap" rel="stylesheet">
+    <link href="https://fonts.googleapis.com/css2?family=Roboto:wght@500&display=swap" rel="stylesheet">
   </head>
   <body>
-    <h1>Catalog</h1>
+    <header class="header">
+      <a href="#home">
+        <img 
+          src="./images/logo.png" 
+          class="header__logo" 
+          alt="moyo"
+        >
+      </a>
+      <nav class="nav">
+        <ul class="nav__list">
+          <li class="links">
+            <a href="#apple" class="links__link_a is-active">Apple</a>
+          </li>
+
+          <li class="links">
+            <a href="#samsung" class="links__link_a">Samsung</a>
+          </li>
+
+          <li class="links">
+            <a href="#smartphones" class="links__link_a">Smartphones</a>
+          </li>
+
+          <li class="links" data-qa="nav-hover">
+            <a href="#laptops" class="links__link_a">Laptops & Computers</a>
+          </li>
+
+          <li class="links">
+            <a href="#gadgets" class="links__link_a">Gadgets</a>
+          </li>
+
+          <li class="links">
+            <a href="#tablets" class="links__link_a">Tablets</a>
+          </li>
+
+          <li class="links">
+            <a href="#photo" class="links__link_a">Photo</a>
+          </li>
+          
+          <li class="links">
+            <a href="#video" class="links__link_a">Video</a>
+          </li>
+        </ul>
+      </nav>   
+    </header>
+    
+    <main class="main">
+      <div class="main__container">
+        <div class="card" data-qa="card">
+          <img
+            src="./images/imac.jpeg"
+            class="card__img"
+            alt="APPLE A1419 iMac 27"
+          >
+          <h2 class="card__title">APPLE A1419 iMac 27" Retina 5K Monoblock (MNED2UA/A)</h2>
+          <p class="card__type">Product code: 195434</p>
+          <div class="card__review-container">
+            <div class="stars stars--4">
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+              <div class="stars__star"></div>
+            </div>
+            <div class="card__review">Reviews: 5</div>
```

Comment:
Use a semantic tag everywhere for text
```suggestion
            <p class="card__review">Reviews: 5</p>
```